packages/devextreme/js/__internal/ui/m_tag_box.ts

```diff
@@ -1058,13 +1087,15 @@ const TagBox = (SelectBox as any).inherit({
       this._tagElements().remove();
     } else {
       const $tags = this._tagElements();
-      const values = this._getValue();
+
+      const selectedItems = this._selectedItems ?? [];
```

Comment:
```suggestion
      const selectedItems = this._fieldRenderData() ?? [];
```

Reply:
tbh Alex's option is more clear to me)

Reply:
i will use `selectedItems` from option(), should be ok in this place, since it's already updated